doc/src/tutorials/biomechanics/biomechanical-model-example.rst

```diff
@@ -0,0 +1,998 @@
+
+.. plot::
+   :format: doctest
+   :include-source: True
+   :context: close-figs
+   :nofigs:
+
+   >>> u_vals = np.array([
+   ...     0.0,  # u1, [rad/s]
+   ...     0.0,  # u2, [rad/s]
+   ...     0.0,  # u3, [rad/s]
+   ...     0.0,  # u4, [rad/s]
+   ... ])
+   ...
+
+   >>> a_vals = np.array([
+   ...     0.0,  # a_bicep, nondimensional
+   ...     0.0,  # a_tricep, nondimensional
+   ... ])
+
+The muscle excitations will also initially be deactivated:
+
+.. plot::
+   :format: doctest
+   :include-source: True
+   :context: close-figs
+   :nofigs:
+
+   >>> e_vals = np.array([
+   ...     0.0,
+   ...     0.0,
+   ... ])
+
+The system equations can be now be numerically evaluated:
+
+.. plot::
+   :format: doctest
+   :include-source: True
+   :context: close-figs
+   :nofigs:
+
+   >>> eval_diffeq(q_vals, u_vals, a_vals, e_vals, p_vals)
+   ([[ 0.00333333 -0.15174161 -0.00109772 -0.00152436]
+    [ 0.19923894  0.31       -0.04923615  0.00996712]
+    [ 0.01743115  0.          0.29585191  0.0011276 ]
+    [ 0.         -0.29256885 -0.0005241  -0.29983226]], [[-0.9121071]
+    [ 0.       ]
+    [-0.       ]
+    [ 0.       ]], [[0.]
+    [0.]])
+
+Simulate the Muscle-actuated Motion
+===================================
+
+Now that the system equations can be evaluated given the state and constant
+values we can simulate the arm and lever's motion with excitation of the two
+muscles. SciPy's ``solve_ivp()`` can integrate the differential equations if we
+provide a function that evaluates them in explicit form, i.e.
+:math:`\dot{\mathbf{x}}=`. We will include a function to excite the muscles but
+set it to zero for the first simulation.
+
+.. plot::
+   :format: doctest
+   :include-source: True
+   :context: close-figs
+   :nofigs:
+
+   >>> def eval_r(t):
+   ...     """Returns the muscles' excitation as a function of time."""
+   ...     e = np.array([0.0, 0.0])
+   ...     return e
+   ...
+   >>> def eval_rhs(t, x, r, p):
+   ...     """Returns the time derivative of the state.
+   ...
+   ...     Parameters
+   ...     ==========
+   ...     t : float
+   ...         Time in seconds.
+   ...     x : array_like, shape(10,)
+   ...         State vector.
+   ...     r : function
+   ...         Function f(t) that evaluates e.
+   ...     p : array_like, shape(?, )
+   ...         Parameter vector.
+   ...
+   ...     Returns
+   ...     =======
+   ...     dxdt : ndarray, shape(10,)
+   ...       Time derivative of the state.
+   ...
+   ...     """
+   ...
+   ...     q = x[0:4]
+   ...     u = x[4:8]
+   ...     a = x[8:10]
+   ...
+   ...     e = r(t)
+   ...
+   ...     qd = u
+   ...     m, f, ad = eval_diffeq(q, u, a, e, p)
+   ...     ud = np.linalg.solve(m, f).squeeze()
+   ...
+   ...     return np.hstack((qd, ud, ad.squeeze()))
+   ...
+
+The system can now be simulated over 3 seconds provided the initial state
+:math:`\mathbf{x}_0` and our function defined above using SciPy's
+``solve_ivp()``.
```

Comment:
Add a note about how we aren't enforcing the holonomic constraint and there will be drift.